pkg/private/zip/build_zip.py

```diff
@@ -168,6 +169,7 @@ def add_manifest_entry(self, entry):
     if entry_type == manifest.ENTRY_IS_FILE:
       entry_info.compress_type = self.compression_type
       # Using utf-8 for the file names is for python <3.7 compatibility.
+      entry_info.external_attr |= UNIX_FILE_BIT << 16  #  DO WE NEED AN MSDOS_DIR_BIT?
```

Comment:
@romanofski  
I'm not sure this is sufficient for your case on Azure. Can you patch in this change and try it?  
Mu suspiciaion is that we need another bit.

Reply:
Yeah will try out. Thanks mate for the quick patch!